src/styles/header.scss

```diff
@@ -0,0 +1,54 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+  font-family: Roboto, Arial, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+
+  --font-color: #00acdc;
```

Comment:
The custom property '--font-color' is defined within the '.header' class, which limits its scope to that class. If you intend to use this variable across multiple classes, consider defining it in a more global scope, such as ':root'.